README.md

```diff
@@ -155,6 +181,8 @@ Make sure to first run `npm install` to install the dependencies required to run
 lein kaocha
 ```
 
+*Note: Running tests requires you to have a [Biotestmine](https://github.com/intermine/biotestmine) running locally. The easiest way to achieve this is by using [intermine_boot](https://github.com/intermine/intermine_boot).*
```

Comment:
this is great, BUT I'd add a mini guide for how to run biotestmine with intermine_boot as well, e.g. 

```bash
pip install intermine_boot
intermine_boot start local
```
then go to localhost:9999/biotestmine to verify it's running. 

(alternatively, maybe make it clearer in intermine boot that the biotestmine is the default mine? either is fine). 

or if I've surmised that wrong.... then the correct instructions instead 😆

Reply:
I don't really want to add instructions to this README since the intermine_boot invocation is bound to change. I think we can make it clearer (in the docs or command) in intermine_boot, or even mention it in the biotestmine README, but things are uncertain now due to being under development. [=